data/raw/buildings_data.csv

```diff
@@ -1,53 +1,53 @@
-minnesota,2914016,9838,5124,8130,4655,6624,2640,3478,82.64,90.85,67.33,51.52,35.35
-mississippi,1507496,5345,2502,2297,1512,1920,1044,1635,42.97,60.43,35.92,41.73,30.59
-missouri,3190076,11309,6260,7204,4875,5824,2969,2627,63.7,77.88,51.5,47.43,23.23
-montana,773199,2032,967,1558,763,1334,498,568,76.67,78.9,65.65,51.5,27.95
-nebraska,1187234,3368,2038,2348,1648,1928,1024,828,69.71,80.86,57.24,50.25,24.58
-nevada,1006278,4955,2747,3193,2175,3290,1287,2364,64.44,79.18,66.4,46.85,47.71
-new_hampshire,577936,2561,1541,2355,1371,1722,669,879,91.96,88.97,67.24,43.41,34.32
-new_jersey,2550308,14804,10631,12971,9560,12084,5813,8863,87.62,89.93,81.63,54.68,59.87
-new_mexico,1037096,3767,2322,3092,2050,2743,1174,2009,82.08,88.29,72.82,50.56,53.33
-new_york,4972497,33711,19471,29912,17359,26120,10561,17967,88.73,89.15,77.48,54.24,53.3
-north_carolina,4678064,18384,11673,6552,6625,5597,4200,4924,35.64,56.75,30.44,35.98,26.78
-north_dakota,568213,1445,674,852,504,779,288,390,58.96,74.78,53.91,42.73,26.99
-ohio,5544032,21255,14987,16308,13048,14427,7909,9335,76.73,87.06,67.88,52.77,43.92
-oklahoma,2159894,7020,4948,4300,3784,3593,2257,2502,61.25,76.48,51.18,45.61,35.64
-oregon,1873786,7043,4990,3402,2759,2624,1863,1059,48.3,55.29,37.26,37.33,15.04
-pennsylvania,4965213,23083,14478,17930,10368,15875,6737,11637,77.68,71.61,68.77,46.53,50.41
-rhode_island,392581,1911,1447,1713,1291,1439,689,884,89.64,89.22,75.3,47.62,46.26
-south_carolina,2299671,9049,7419,2383,3519,2471,2416,2142,26.33,47.43,27.31,32.57,23.67
-south_dakota,661311,1548,698,1064,545,878,318,447,68.73,78.08,56.72,45.56,28.88
-tennessee,3212306,11863,7793,4482,4597,4385,3112,2088,37.78,58.99,36.96,39.93,17.6
-texas,10678921,43108,28015,16557,16627,16206,10508,12364,38.41,59.35,37.59,37.51,28.68
-utah,1081586,4227,3047,3721,2833,3364,1619,999,88.03,92.98,79.58,53.13,23.63
-vermont,351266,1350,500,1285,459,909,229,454,95.19,91.8,67.33,45.8,33.63
-virginia,3079351,14203,7374,6284,4591,5345,2811,2521,44.24,62.26,37.63,38.12,17.75
-washington,3128258,12248,6942,5251,3497,4176,2388,1782,42.87,50.37,34.1,34.4,14.55
-west_virginia,1055625,3653,2326,2003,1644,1640,959,729,54.83,70.68,44.89,41.23,19.96
-wisconsin,3173347,10944,6627,9266,6193,7325,3717,3854,84.67,93.45,66.93,56.09,35.22
-wyoming,386518,1104,491,855,406,745,245,299,77.45,82.69,67.48,49.9,27.08
+state,buildings,nrelRes,nrelCom,resNonElHeating,comNonElHeating,resNonElWaterHeating,resNonElWaterHeating,resNonElRange,pctResNonElHeating,pctComNonElHeating,pctResNonElWaterHeating,pctComNonElWaterHeating,pctResNonElRange,pctRes,pctCom,weightedRes,weightedCom,weightedFossilBuildingsPct,weightedEleBuildingsPct
+alaska,111042,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+hawaii,252908,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+united_states,30099920,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+alabama,2455168,9125,5286,3136,2894,3010,2069,2481,34.37,54.75,32.99,39.14,27.19,0.633196863507043,0.366803136492957,33.1275,50.28554,39.42,60.58
```

Comment:
This seems like an excessive level of precision - if it's not too hard I'd suggest trimming to at most three numbers after the decimal if this is already a percentage out of 100, and 5 if not (e.g. 50.234% or 0.50234). Otherwise I worry someone will think this data is more precise than it really is.

Reply:
Agreed. Only the final two columns (2 decimal precision) make it to the frontend, but this is definitely good practice. Resolved in b122ca9